### PR TITLE
Update release notes 536

### PR DESCRIPTION
**Pull request by tst-sbayard**
Updated new features and list of resolved issues for the 5.3.6 release notes. To review, check the list of resolved issues with a milestone of 5.3.6 and see if there are any that should have been included.

### CODE REVIEW

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst [hunk 1]
```diff
@@ -12,14 +12,10 @@ Release notes: Open GEE 5.3.6
...
-
-      In 5.3.5, we moved documentation, examples, and tutorials into a
-      separate RPM called opengee-extra. This mainly includes the
-      documentation, ExampleSearch, and Fusion Tutorial files.  You
-      can choose whether to install opengee-extra depending on if you
-      need those items.
+      In 5.3.6, a ``--secure`` option was added for ``geconfigureassetroot``
+      and ``geupgradeassetroot`` that removes world-writable permissions from
+      special files. This helps Open GEE perform properly in strict security
+      environments. ``--secure`` is not the default setting.
```
Comment (tst-lsavoie) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst:18]: I would reword the last sentence to something like
```
``--secure`` is disabled by default.
```

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst [hunk 2]
```diff
@@ -44,28 +40,30 @@ Release notes: Open GEE 5.3.6
...
          * - Number
            - Description
-           - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1851
+           - Open GEE does not function properly on systems with a restrictive umask
+         * - 1850
+           - Fusion should include an option to set secure permissions for special folders
```
Comment (tst-lsavoie) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst:51]: This is the same as the change listed under "New Features", right? I don't think it needs to be both places.

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst [hunk 3]
```diff
@@ -44,28 +40,30 @@ Release notes: Open GEE 5.3.6
...
+         :widths: 35 65
          :header-rows: 1
 
          * - Number
            - Description
-           - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1851
```
Comment (tst-lsavoie) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst:48]: It looks like we generally list issues in reverse order of issue number. Can we sort these in the same order?

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst [hunk 4]
```diff
@@ -12,14 +12,10 @@ Release notes: Open GEE 5.3.6
```
Comment (tst-lsavoie) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst:12]: I think we should remove the above line ("Open GEE 5.3.6 is currently in development"). Technically it's still true but we don't want to have to make another commit after we finish testing.

File: docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst [hunk 5]
```diff
@@ -44,28 +38,28 @@ Release notes: Open GEE 5.3.6
...
-           - Add description
-           - Add resolution
+         * - 1851
+           - Open GEE does not function properly on systems with a restrictive umask
+         * - 1806
+           - Confusing "geserver.service changed on disk" message after uninstalling Fusion or Server
+         * - 1800
+           - Upgrade from 5.3.4 to 5.3.5 leaves a non-working ExampleSearch
+         * - 1785
+           - Imagery Projects - Transparent areas appears as black
```
Comment (tst-lsavoie) [inline docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst:53]: Minor: just noticed that there's a grammar problem here. Should be `Transparent areas appear as black` (no "s" in "appear").

**Review by cuppster — APPROVED** [review]
